sormas-api/src/main/java/de/symeda/sormas/api/EntityDto.java

```diff
@@ -46,6 +46,7 @@
 	public static final String CREATION_DATE = "creationDate";
 	public static final String CHANGE_DATE = "changeDate";
 	public static final String UUID = "uuid";
+	public static final String NO_LAST_SYNCED_UUID = "NO_LAST_SYNCED_UUID";
```

Comment:
Why not use null and/or an empty string?

Reply:
Empty or null Strings are not compatible with the REST API here.